compiler/vm/vmjit.nim

```diff
@@ -38,6 +38,10 @@ export VmGenResult
 const
   debugEchoCode* = defined(nimVMDebugGenerate)
 
+when debugEchoCode:
```

Comment:
Preferably should be moved to the top of the file as well.

Reply:
That's only possible if the usage of `debugEchoCode` here is replaced with `defined(nimVMDebugGenerate)`, which would mean that the imports are guarded by a different expression than the dependents of the import, rendering `debugEchoCode` somewhat useless.

I can move the `debugEchoCode` constant to `vmdef` instead.

Reply:
Just remove `debugEchoCode` altogether and use defines as the rest of debugging utils does